app/views/layouts/ats/application.html.slim

```diff
@@ -26,6 +26,13 @@ html
     = stylesheet_link_tag "ats", media: "all", "data-turbo-track": "reload"
     = javascript_include_tag "ats", "data-turbo-track": "reload"
     = javascript_include_tag "https://www.gstatic.com/charts/loader.js"
+    - if ENV["CLARITY_KEY"].present?
```

Comment:
Nitpick: it's not a good pattern to use env on the view, what do you thing about fetching it to some const on the ApplicationController and use const here?